.github/workflows/docs.yml

```diff
@@ -0,0 +1,72 @@
+name: doxygen gh-pages
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    branches: [ develop ]
+    paths-ignore:
+      - '.github/CODEOWNERS'
+      - '.github/FUNDING.yml'
+      - 'doc/**'
+      - 'release_docs/**'
+      - 'ACKNOWLEDGEMENTS'
+      - 'COPYING**'
+      - '**.md'
+
+# Using concurrency to cancel any in-progress job or run
+concurrency:
+  group: ${{ github.workflow }}-${{ github.sha || github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Sources
+        uses: actions/checkout@v3
+
+      - name: Install Dependencies
+        run: |
+          sudo apt update -y
+          sudo apt install -y libunwind-dev
+          sudo apt-get install -y graphviz
+          sudo apt-get install -y --no-install-recommends doxygen
+          sudo apt-get install -y cmake
+          
+      - name: Build Develop Documentation
+        run: |
+          git clone https://github.com/HDFGroup/hdf5.git
+          cd hdf5
+          mkdir build
+          cd build
+          cmake -DHDF5_BUILD_DOC:BOOL=ON ..
+          make doxygen
+
+      - name: Deploy Develop
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          publish_dir: ./hdf5/build/hdf5lib_docs/html
+          destination_dir: develop
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+
+      - name: Build 1.14.1 Documentation
```

Comment:
Why build 1.14.1 docs in a develop PR? Also, this file will need to be fixed every time 1.14 is released.

Reply:
[Get branch name · Actions · GitHub Marketplace]
(   https://github.com/marketplace/actions/get-branch-name   )

Reply:
@byrnHDF Thanks for reviewing! This PR was a quick demo patch for @brtnfld to show him how both develop & 1.14.1-2 Doxygen documentation should look like on GH pages since S3 docs were out of sync.